frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -22,14 +24,36 @@ class NewPassword extends Component {
     this.setState({ [event.target.name]: event.target.value });
   };
 
+  handleSubmit = (event) => {
+    const { token, history } = this.props;
+    event.preventDefault();
+    const { password, confirmPassword } = this.state;
+    if (password !== confirmPassword) {
+      this.setState({ error: "Passwords don't match" });
+    } else {
+      resetPassword(token, password)
+        .then((response) => {
+          if (response) {
+            history.push('/login');
+          }
+        })
+        .catch((error) => {
+          if (error) {
+            this.setState({ error: 'Error' });
+            console.log('Error response', error.response.data.errors); // eslint no-console
```

Comment:
@vhoof @H12 I felt like leaving it more general to have a console log that gives us the error response was better for multiple types of errors rather than making it specific to the the reset password token error. This way if the error is unrelated to the password reset token, we will still get specific information for that particular error.